### DISCUSSION TITLE
Missing configuration "Win64"

### DISCUSSION THREAD

**Discussion by petzenred** [microsoft/AirSim#3898] [Support Q&A] [answered]
Hi, to preface this question I've never used the Unreal Engine or AirSim before, so apologies if this question has a very obvious answer I haven't been able to find. 

I've been setting up AirSim using the directions from Microsoft.github.io/AirSim and am on a Windows 10 computer. Specifically on this page: 
https://microsoft.github.io/AirSim/unreal_blocks/ 
you're told to make sure the build configuration is set to DebugGame_Editor and Win64 before running; but I don't have that option. I have DebugGame_Editor|Win32 and DebugGame_Editor|x64, but no DebugGame_Editor|Win64. I tried running it with x64, but the build failed, and in the solution explorer I can see the Intermediate<Build folder only has a Win64 folder (which I assume is why it wouldn't build). What might have caused this, and how can I fix it? 
Thanks so much

**Comment by zimmy87** [comment, marked as answer]
win64 is the name of the platform configuration in the Blocks.sln file. x64 is the name of the platform configuration in the Blocks.vcxproj file. The win64 solution configuration should be referencing the x64 project configuration. This appears to just be a difference in naming conventions, it shouldn't result in a build failure. What errors are you getting when your build fails? Also where in visual studio are you selecting the build configuration? You should only see options for the solution configuration in the toolbar dropdowns.

**Reply by petzenred** [reply]
Hi, thank you for responding and sorry for the delay, my internet was out for a few days because of a storm. 
When I reopened VS the day after I posted this comment to try and work on it, the build worked without a problem and has since, so I have no idea why the build was failing the first day. All's well that ends well, though confusingly, I guess! If anyone in the future is reading this post, try rebooting the computer - for some reason that fixed this issue for me. 
I'm afraid I don't remember what the build failure was off of the top of my head, but I was definitely choosing the configuration from the dropdown menu in the toolbar. 
Thanks again for responding.
